recipes/kantoku/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "kantoku" %}
+{% set version = "0.18.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/kantoku-{{ version }}.tar.gz
+  sha256: 6d25065f9da68949aa8181c59151a4670230720ac2b054059fcf6daf1f618740
+
+build:
+  entry_points:
+    - circusd = circus.circusd:main
+    - circusd-stats = circus.stats:main
+    - circusctl = circus.circusctl:main
+    - circus-top = circus.stats.client:main
+    - circus-plugin = circus.plugins:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - flit-core >=3.4,<4
+    - pip
+  run:
+    - python >={{ python_min }}
+    - psutil
+    - pyzmq >=17.0
+    - tornado >=5.0.2
+    - __unix  # [unix]
+    - __win   # [win]
```

Comment:
Do you want to create a package for Windows here? I thought that it won't work there, so we can make a noarch that will only work on Linux and macOS instead, right?

Reply:
A [quick read upstream](https://github.com/search?q=repo%3Abentoml%2Fkantoku+windows&type=code) gave me the impression that it does work on Windows in a limited capacity. So you were correct before. Do you mind re-adding Windows?

Reply:
@ocefpaf - Done, Please take a look. thank you!